README.md

```diff
@@ -66,6 +74,7 @@ To run the tests, run
 poetry shell
 poetry run pytest
 ```
+Code in the test directory is almost identical to the above example code.
```

Comment:
Confused about this statement. What does this mean?

Reply:
test case is basically running the inference example provided in the README.md

Reply:
Ah ok. Can delete this sentence.